EMRM.py

```diff
@@ -247,7 +295,14 @@ def alpha_beta_policy(self, alpha, beta, max_time):
                 return(self.reader.vocab.wlen + 2)
 
 class OneBlock:
+    """ Class of a block of trials.
+
+    Attributes:
+        reader: A OneVirtualReader with a vocabulary and some visual properties.
+        trial_list: A list of trial information stored in dictionaries. Each dictionary has a `word` and a `scanpath` key.
```

Comment:
It's not very clear to me what trial_list looks like. An example might help?

Reply:
example added